HelpSource/scdoc.css

```diff
@@ -611,18 +626,20 @@ ul.tree li {
 }
 
 ul.tree li:last-child {
- background: #fff url(images/lastnode.png) no-repeat;
+ background: var(--sc-bg-color) url(images/lastnode.png) no-repeat;
 }
 
+/* is this still used? */
 :target {
-    border-bottom: 6px solid rgba(255,255,0,0.4);
+    border-bottom: 6px solid var(--sc-bg-warning);
     padding-top:20px;
     margin-top:-20px;
 }
 
 a.undoc {
-    color: #88b;
+    color: var(--sc-text-color-lighter);
```

Comment:
now that we set opacity to 50%, could this be the same color as the other links? removing the color definition would be enough to inherit their color.